homeassistant/components/shelly/switch.py

```diff
@@ -60,6 +60,11 @@ def is_on(self) -> bool:
 
         return self.block.output
 
+    @property
+    def device_state_attributes(self) -> dict:
+        """Additional switch attributes."""
+        return {"appliance_type": self.wrapper.device.settings.get("appliance_type")}
```

Comment:
We shouldn't include this. A user can already see what kind of appliance type it is by seeing it being a light or a switch entity?

Reply:
Correct; but when the entity is a switch it can be of many appliance types ( domestic appliance, fan, etc. ) and the attribute would be useful to group those.

Reply:
It is a secondary categorization from upstream, that has no effect on our end. Therefore I agree with balloob that this doesn't belong in our state engine.

Reply:
So where should this info be placed on your opinion ?

Reply:
Nowhere. The information used to determine platforms and behaviors inside Home Assistant on a lower level.

From a user perspective, naming, categorizing and placing devices in groups and such, are handled by the user in Home Assistant itself and is not fed by data as presented here.

Reply:
Thx for sharing your point of view.

I don't want to create a case for such minor issue, but how can I group by type if I lack this info at all ?

Simone

Reply:
There is no group by type in the HA world. Groups in Home Assistant (or areas for that matter) have no relation to the `appliance_type` of Shelly? Am I missing something?

Reply:
After speaking with Frenk, I got his point: he told me that this attribute will be used only for a custom group-ing approach and this is not part of the greater / bigger whole of Home Assistant should not be considered.

I think this is the real point, so I'm going to close this PR.